inc/3rd-party/hosting/litespeed.php

```diff
@@ -0,0 +1,76 @@
+<?php
```

Comment:
@crystinutzaa Is this a new file for this PR? Or a carryover from a merged commit?

Reply:
It's a carry over. It was deleted and moved to this file:
https://github.com/wp-media/wp-rocket/blob/master/inc/classes/subscriber/third-party/Hostings/class-litespeed-subscriber.php
I will delete it.